src/org/usfirst/frc/team2585/AutonomousCommand.java

```diff
@@ -0,0 +1,11 @@
+package org.usfirst.frc.team2585;
+
+/**
+ * A command to be passed to the autonomous executor
+ */
+public abstract class AutonomousCommand {
+	/**
+	 * @param timeElapsed time passed since auton started
```

Comment:
Since this only has a single abstract method, this class should be an interface instead.